### DISCUSSION TITLE
Phonenumbers (WorkPhone) in the Filter web part

### DISCUSSION THREAD

**Discussion by arkJK** [microsoft-search/pnp-modern-search#3090] [Q&A] [answered]
Hello Guys,

i could not find a topic related to this and decided to open a new one. 
I want to create a intern people directory with a possibilty to look after departments, last name and phone numbers. 
Department and last name are working as intended but it seems that i can not show the phone numbers. 
I modified properties to try it with other number columns but with the same result. In the pnp - results web part the WorkPhone is shown as intended.
Is this even possible?

**Comment by kasperbolarsen** [comment]
I am not sure I understand the purpose. As each phone number is unique it is not really optimal as a refiner, unless the organization is very small like less that 100 employees.

**Reply by arkJK** [reply]
It is really small. It is just intern and is for 70 employees.

**Comment by kasperbolarsen** [comment, marked as answer]
OK, you will have to map People:Workphone to a refinablestring, as the WorkPhone managed Property isn't refinable:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/8ba1e8b5-9ec2-4502-bf32-7f56f50ecf52)

**Reply by arkJK** [reply]
Thank you @kasperbolarsen. I´ll try.
